libsweep/src/sweep.cc

```diff
@@ -218,11 +218,25 @@ void sweep_device_stop_scanning(sweep_device_s device, sweep_error_s* error) {
     return;
   }
 
-  // Wait some time, for the device to register the stop cmd and stop sending data blocks
+  // Wait some time for a few reasons:
+  // 1. allow time for the device to register the stop cmd and stop sending data blocks
+  // 2. allow time for slower performing devices (RaspberryPi) to buffer the stop response
+  // 3. allow a grace period for the device to perform its logging routine before sending a second stop command
```

Comment:
As said in the ticket, if we can just make the timeout below a bit longer I would go for that. Even hundreds of milliseconds are fine for construction if it makes things easier for us.